public/stylesheets/main.css

```
@@ -539,9 +539,19 @@ Colors
     color: #00234d !important;
 }
 
+/* from --blue: #0053b3;
+ * https://github.com/etalab/template.data.gouv.fr/blob/master/src/css/base/theme.css#L5
+ *
+ * Usage:
+ * - links
+ */
+.--data-gouv-text--blue {
```

```suggestion
.aplus-color-text--blue {
```
J'ai appelé aplus-color les couleurs officiels